config/initializers/session_store.rb

```diff
@@ -1,4 +1,4 @@
-if !Rails.env.development? && !Rails.env.production?
+if ENV['RAILS_USE_MEMORY_STORE'] || (!Rails.env.development? && !Rails.env.production?)
```

Comment:
specifically, require of 'config/environment' tries to connect to memcached and fails.